css/src/components/form/textarea.scss

```diff
@@ -145,4 +148,15 @@ $textarea-divider-radius: 0.5rem !default;
 		gap: $textarea-form-gap;
 		height: 100%;
 	}
+
+	.textarea-form-help {
+		.help {
+			display: flex;
```

Comment:
Suggestion only, should allow you to remove the `icon` rule.
```css
.help {
    display: flex;
    align-items: center;
    gap: .375em; // or whatever value
 }
    ```

Reply:
got it to work with npm link 

![image](https://github.com/user-attachments/assets/27ef6d51-9151-4b7d-8663-cb27ab33be10)